Bio/SubsMat/__init__.py

```diff
@@ -333,6 +342,16 @@ def print_mat(self, f=None, format="%4d", bottomformat="%4s",
                       " please use\n"
                       " print(mat.format(<arguments>)",
                       BiopythonDeprecationWarning)
+        if factor == 1:
+            mat = self
+        else:
+            mat = factor * self
+            warnings.warn("""
+Instead of
+mat.print_mat(..., factor, ...)
+please use
+mat2 = factor * mat
+print(mat2.format(...))""", BiopythonDeprecationWarning)
```

Comment:
Same comment as about about the triple-quotes

Reply:
Fixed